test/musica/CMakeLists.txt

```diff
@@ -5,8 +5,9 @@ FetchContent_Declare(musica
   GIT_REPOSITORY https://github.com/NCAR/musica.git
   GIT_TAG        $ENV{MUSICA_GIT_TAG}  # Set by docker
 )
+message(STATUS "Using MUSICA commit: $ENV{MUSICA_GIT_TAG}")
 
-set(MUSICA_BUILD_C_CXX_INTERFACE OFF)
+set(MUSICA_BUILD_C_CXX_INTERFACE ON)
```

Comment:
oh I guess to test `Dockerfile.musica.no.install`?

Reply:
TUVx, is only included when we build the c/c++ interface. I think it's because we eventually want to move tuv out of fortran and into c++. To hide that conversion, we need to include it in musica now as if it were in c++.

Reply:
Ah that makes sense, thank you